node/src/bin/manager.rs

```diff
@@ -388,6 +388,25 @@ pub enum IndexCommand {
         )]
         method: String,
     },
+    /// Lists existing indexes for a given Entity
+    List {
+        /// The id of the deployment
+        #[structopt(empty_values = false)]
+        id: String,
+        /// The Entity name, in camel case.
+        #[structopt(empty_values = false)]
+        entity: String,
```

Comment:
It would be nice if this allowed both entity type names and table names, so that I can say both (and mix and match)
```
graphman index list QmRfCDkbkACHdidsCKADjUDf6S3mMjS381xyzotJDsbqGq LiquidityPosition
``` 
and
```
graphman index list sgd1234 liquidity_position
```

For the deployment lookup, you can use `node::manager::Deployment::lookup` which can be resolved to a `DeploymentLocator` using `Deployment.locator()`

Translating the second argument could just try to look up the table assuming that `entity` is a type name, and if that fails look it up as the SQL name of a table.